include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -503,7 +503,7 @@ typedef struct OrtMIGraphXProviderOptions {
 */
 typedef struct OrtOpenVINOProviderOptions {
 #ifdef __cplusplus
-  OrtOpenVINOProviderOptions() : device_type{}, enable_vpu_fast_compile{}, device_id{}, num_of_threads{}, use_compiled_network{}, blob_dump_path{}, context{} {}
+  OrtOpenVINOProviderOptions() : device_type{}, enable_vpu_fast_compile{}, device_id{}, num_of_threads{}, use_compiled_network{}, blob_dump_path{}, context{}, enable_opencl_throttling{} {}
```

Comment:
using c api, we're supposed to maintain backwards compatibility (with abi)
i.e. a user can drop in a new version of onnxruntime shared lib (built with openvino ep)
and it would work. But here, applications built using earlier ort version would be using a version of OrtOpenVINOProviderOptions struct that did not contain this new field.
do you know if openvino ep users rely on abi? or will they always recompile their application with latest ort headers/library.
This is why we have moved towards using opaque structs to avoid this issue:
see https://github.com/microsoft/onnxruntime/pull/7808 as example.
we create a V2 version of struct, which the user does not directly manipulate. they use api's to construct and pass the struct in as option. the api's use key/value strings for EP options.